packages/cli/src/commands/upgrade.ts

```diff
@@ -46,7 +46,7 @@ upgradeCommand
   .name('upgrade')
   .description('Upgrade typescript dev-dependency with compilation insights and auto-fix options')
   .argument('[basePath]', 'Path to directory contains tsconfig.json', '.')
-  .option('-b, --build <beta|next|latest>', 'typescript build variant', 'beta')
+  .option('-b, --build <beta|next|latest|rc>', 'typescript build variant', 'beta')
```

Comment:
We also need to add some artificial names for `latest of latest/beta/rc` and `latest of latest/beta/rc/dev` (currently beta-4.9.2, dev-4.9.1 ;) )

Just realized... we can specify them as comma separated list and use biggest number available version.

Reply:
I think `latest-beta` is the most accurate and should be the default. Otherwise we should fetch whichever tag is explicitly being passed